src/components/box/box.stories.mdx

```diff
@@ -273,27 +84,7 @@ Use the `overflowWrap` prop to apply `overflow-wrap: break-word` to the content
 <Canvas>
   <Story
     name="overflow wrap"
-    parameters={{ info: { disable: true }, chromatic: { disable: true } }}
-  >
-    <div style={{ display: "inline-flex" }}>
-      <div
-        style={{
-          border: "solid 1px #00815D",
-          width: "min-content",
-          marginRight: "20px",
-        }}
-      >
-        <Box p={1} overflowWrap="break-word" width="100px">
-          WithOverflowWrap
-        </Box>
-      </div>
-      <div style={{ border: "solid 1px #00815D", width: "min-content" }}>
-        <Box p={1} width="100px">
-          WithoutOverflowWrap
-        </Box>
-      </div>
-    </div>
-  </Story>
+    parameters={{ chromatic: { disable: true } }} story={stories.OverflowWrap} />
```

Comment:
Why is this story disabled for Chromatic? I can see it was previously, but what is the reason?